### PR TITLE
FA-9: Java Modules - Jar and Maven Based

### PR DESCRIPTION
**Pull request by username-david**

### CODE REVIEW

File: magnolia-voting-app/pom.xml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      <artifactId>magnolia-core</artifactId>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.7.0</version>
```
Comment (llamkhan) [inline magnolia-voting-app/pom.xml]: don't need to have version here, lets maven handle version

File: magnolia-voting-app/pom.xml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>info.magnolia.bundle</groupId>
+        <artifactId>magnolia-bundle-parent</artifactId>
+        <version>${magnoliaBundleVersion}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
```
Comment (llamkhan) [inline magnolia-voting-app/pom.xml]: do we really need this bundle declaration?

File: magnolia-voting-app/src/main/java/com/example/voting/setup/VotingModuleVersionHandler.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+ * by registering "deltas" to maintain the module's configuration, or other type of content.
+ * If you don't need this, simply remove the reference to this class in the module descriptor xml.
+ *
+ * @see info.magnolia.module.DefaultModuleVersionHandler
+ * @see info.magnolia.module.ModuleVersionHandler
+ * @see info.magnolia.module.delta.Task
+ */
+public class VotingModuleVersionHandler extends DefaultModuleVersionHandler {
+
+}
```
Comment (llamkhan) [inline magnolia-voting-app/src/main/java/com/example/voting/setup/VotingModuleVersionHandler.java]: For classes which haven't implemented yet, can we remove?

File: magnolia-voting-app/pom.xml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    </resources>
+  </build>
+
+  <repositories>
+    <repository>
+      <id>magnolia.public</id>
+      <url>https://nexus.magnolia-cms.com/content/groups/public</url>
+      <snapshots>
+      </snapshots>
+    </repository>
```
Comment (llamkhan) [inline magnolia-voting-app/pom.xml]: Remove this configuration, it's locally used at the moment. Don't need to specify magnolia remote repository yet

**Review by llamkhan — APPROVED** [review]

**Review by ThuyTo — APPROVED** [review]

**Pull request merged**